cli/src/alluxio.org/cli/cmd/quorum/elect.go

```diff
@@ -0,0 +1,57 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package quorum
+
+import (
+	"github.com/spf13/cobra"
+
+	"alluxio.org/cli/env"
+)
+
+var Elect = &ElectCommand{
+	BaseJavaCommand: &env.BaseJavaCommand{
+		CommandName:   "elect",
+		JavaClassName: "alluxio.cli.fsadmin.FileSystemAdminShell",
+		Parameters:    []string{"journal", "quorum"},
+	},
+}
+
+type ElectCommand struct {
+	*env.BaseJavaCommand
+
+	ServerAddress string
+}
+
+func (c *ElectCommand) Base() *env.BaseJavaCommand {
+	return c.BaseJavaCommand
+}
+
+func (c *ElectCommand) ToCommand() *cobra.Command {
+	cmd := c.Base().InitRunJavaClassCmd(&cobra.Command{
+		Use:   Elect.CommandName,
+		Short: "Transfers leadership of the quorum to the specified server",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return c.Run(nil)
+		},
+	})
+	// TODO: to be consistent with other quorum commands, should this have a domain flag?
```

Comment:
The bash version of the command does not take a `-domain` parameter, and only operates on the `MASTER` domain. I think the go version of the command should work the same.

Reply:
i understand that's how the bash version works and not aiming to change anything for this PR, but for the future, i'd like to know if my statement is true which implies this command should also be domain specific. otherwise someone would wonder if the `elect` command applies to all domains which also makes no sense, thus leading to confusion

Reply:
Maybe it's worth adding a domain parameter in the future to keep things consistent. That would require implementing further logic on the Java side, but not a huge deal.

Reply:
i'd like to add it to the go side to maintain consistency even though it will not be passed along to the java command., so i'd be adding a required flag `--domain` with `MASTER` being the only accepted value. it sets the stage for future changes without any significant impact besides forcing the user to know which domain they are triggering (which i say is a good thing)

Reply:
Sounds good!